src/Extension/FluentVersionedExtension.php

```diff
@@ -185,6 +185,8 @@ protected function augmentSQL(SQLSelect $query, ?DataQuery $dataQuery = null)
                 break;
             // Return all version instances
             case 'archive':
+            case 'archive_only':
+            case 'removed_from_draft':
```

Comment:
These were introduced in https://github.com/silverstripe/silverstripe-versioned/pull/489